### PR TITLE
Various cleanups and fixes to Lt

### PR DESCRIPTION
**Pull request by eric-wieser**
See commit messages for details.

### CODE REVIEW

File: galgebra/lt.py [hunk 1]
```diff
@@ -242,19 +233,21 @@ def __init__(self, *args, ga, f=False, mode='g'):
...
-                        raise ValueError(str(mat_rep) + ' is not supported for Lt definition\n')
-            else:
-                raise ValueError(str(mat_rep) + ' is not supported for Lt definition\n')
+            if F(a + b) != F(a) + F(b):
+                raise ValueError('{} is not linear'.format(F))
+            self.lt_dict = {}
+            for base in self.Ga.basis:
+                out = F(mv.Mv(base, ga=self.Ga))
+                if not out.is_vector():
+                    raise ValueError('{} must return vectors'.format(F))
```
Comment (eric-wieser) [inline galgebra/lt.py:247]: Frankly this restriction seems unnecessary (and forbids linear transformations with a basis in the null space), but I don't want to scope creep too much here

**Comment by eric-wieser** [comment]
Failure is unsurprisingly #441

**Review by utensil — APPROVED** [review]
